src/pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,213 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import './Signup.scss';
+
+import google from '../../assets/images/google.png';
+import facebook from '../../assets/images/facebook.png';
+import github from '../../assets/images/github.png';
+import girl from '../../assets/images/girl.svg';
+import man from '../../assets/images/man.jpg';
+
+import { signup } from '../../actions/signupActions';
+
+export class Signup extends Component {
+  constructor(props) {
+    super(props);
+
+    this.form = React.createRef();
+
+    this.state = {
+      toggled: false,
+      user: {},
+    };
+  }
+
+  onSubmit() {
+    const { user } = this.state;
+    const { history } = this.props;
+
+    this.props.signup(user).then(res => {
+      if (res) {
+        history.push('/');
+      }
+    });
+  }
+
+  organization() {
+    return (
+      <div
+        className="organization col-md-6 col-sm-12 "
+        style={{ backgroundImage: `url(${man})` }}
+      >
+        <div className="wrapper h-100">
+          <div className="title">Register an organization</div>
+          <div className="subtitle">Create an account for your entreprise</div>
+          <div
+            className="register"
+            onClick={() => {
+              this.setState({ ...this.state, toggled: true });
```

Comment:
@gracelungu Extract it to a function instead of changing the state from the view